libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/media/rest_v11_sites_158396482_media_294.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
This file isn't needed anymore since you change to using an image from the library.

Reply:
🤔I was thinking it's worth keeping in case this path in another test or no?

Reply:
We _could_ keep it but I tend to favour keeping things as lean as possible. If it goes unused indefinitely, that's just tech debt. I am open to keeping it though if you think it would be genuinely useful.

Reply:
Hmm ok I have no plans for it for the next few sprints so I'll remove and revert if the use case comes up 👍

Reply:
@JavonDavis What do you think of doing one image selected from the device and one selected from the media library (as in the WPiOS basic publish test), to exercise both parts of the media picker?

Reply:
This can work but I initially opted against this just to keep the tested focus on the editor, but I'm totally fine with doing it to allow us to cover the media picker as well.

Reply:
I think it's worth doing, if there aren't any complications, since that action (uploading media from the device) is a valuable part of the app and editor flow.

Reply:
👍 I'll update the test to cover that!

Reply:
Done

Reply:
I tested with this change (select image from device instead of library) and realized that my emulator doesn't come pre-loaded with device media, so the test can't select any photos to upload. I didn't find any solutions for that (aside from manually adding media to the emulator); do you know of any?

If not, we could switch back to selecting from the media library or try taking a photo instead.

Reply:
> do you know of any? 

No I don't know of a way to pre load images onto the emulator, maybe there could be some kind of pre setup script to upload an image via `adb` but I don't think that would work for a Ci tool, I'll revert back to just using the media library image for now

Reply:
Can we remove the redundant mocks since we have moved back to the library approach?